experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -346,6 +349,10 @@ def resolve_conclusion_recommendations(self, info):
             NimbusExperiment.ConclusionRecommendation
         )
 
+    def resolve_population_sizing_data(self, info):
+        sizing_data = cache.get(SIZING_DATA_KEY)
+        return sizing_data.json() if sizing_data else "Sizing data not available."
```

Comment:
Wait if we return a non JSON parseable string here then the frontend has to distinguish this as a constant? Why not just return null here?